src/models/Tags.ts

```diff
@@ -1,6 +1,21 @@
+export interface TagCategory {
+    tag: string;
+    label: string;
+    checked?: boolean;
+}
+type PossibleTagValueValues = "0" | "1";
+interface TagsObject<T> {
+    [key: string]: T;
+}
+export type TagsObjectWithStrings = TagsObject<string>;
```

Comment:
I think we ended up not using `TagsObjectWithStrings` anywhere, can be safely deleted.

Reply:
Done.